### PR TITLE
[doc/keras] incorrect comment in `__init__` of `tf.keras.layers.AveragePooling1D` #27633

### PR DESCRIPTION
**Pull request by shashvatshahi1998**
#27633

### CODE REVIEW

**Comment by mihaimaruseac** [comment]
Linking [to this comment](https://github.com/tensorflow/tensorflow/pull/27637#issuecomment-480879996) to make PRs look clean and easier to review and merge. Please follow

**Review by mihaimaruseac — APPROVED** [review]
Approved, but since the PR is not clean it might cause issues downstream. Please make clean PRs in the future

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]
This changes `MaxPooling`, not `AveragePooling`

**Comment by shashvatshahi1998** [comment]
@mihaimaruseac  now I have corrected it.

**Comment by mihaimaruseac** [comment]
#27633 also mentions `input_shape` and `output_shape` lists being badly formatted. Can you please make a PR for those too (separate from this one and not including all of these unrelated changes)?

**Review by mihaimaruseac — DISMISSED** [review]

**Comment by shashvatshahi1998** [comment]
Input shape:
    - If `data_format='channels_last'`:
      3D tensor with shape `(batch_size, steps, features)`.
    - If `data_format='channels_first'`:
      3D tensor with shape `(batch_size, features, steps)`.

  Output shape:
    - If `data_format='channels_last'`:
      3D tensor with shape `(batch_size, downsampled_steps, features)`.
    - If `data_format='channels_first'`:
      3D tensor with shape `(batch_size, features, downsampled_steps)`.
@mihaimaruseac  so you want me to edit this input and output shape description in average_pooling function.

**Comment by mihaimaruseac** [comment]
In a __different PR__, if you do it. If you don't do it, that's also ok, just don't close #27633` then.

**Comment by mihaimaruseac** [comment]
You will have to resolve conflicts before this can be merged.

**Comment by shashvatshahi1998** [comment]
@mihaimaruseac  removed the merge conflicts.

File: tensorflow/python/util/compat.py [hunk 1]
```diff
@@ -118,27 +118,7 @@ def path_to_str(path):
...
-  'corpus' # Linux OS
-  >>> tf.compat.path_to_str('./.././Corpus')
-  './.././Corpus' # Linux OS
-  >>> tf.compat.path_to_str(Path('./.././Corpus'))
-  '../Corpus' # Linux OS
-  >>> tf.compat.path_to_str(Path('./..////../'))
-  '../..' # Linux OS
-
-  ```
+
```
Comment (mihaimaruseac) [inline tensorflow/python/util/compat.py]: This change is unrelated and just undoes your previously merged PR.

I suggest we close this one and start a new PR from a fresh fork, making it a clean PR.

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]
You're undoing previous PR now.

**Comment by shashvatshahi1998** [comment]
ok let's close this PR

**Comment by mihaimaruseac** [comment]
Please reopen from a new fork, as a clean PR.

**Pull request closed without merging**